BigriverBookstore-apple/Model/Book.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Book.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Spine
+
+class Book:Resource {
+    
+    var title:String?
+//    var datePublished:Date?
```

Comment:
@c-Viorel I think is better to avoid dead code / commented code to be merged on the master. If we don't need the datePublished yet, let's remove it and add it on the pr when we start using it.